zenodo/modules/records/config.py

```diff
@@ -160,3 +160,9 @@
 
 
 """
+
+ZENODO_FUNDER_PROGRAM_MATCHES = {
```

Comment:
```suggestion
ZENODO_RECORDS_FUNDER_PROGRAM_MATCHES = {
```

With the `ZENODO_RECORDS_` prefix you'll get it automatically loaded also via the extension (this solves the issue of providing a default below).